src/jit/codegencommon.cpp

```diff
@@ -4578,7 +4587,10 @@ void CodeGen::genCheckUseBlockInit()
 
         if (varDsc->lvMustInit && varDsc->lvOnFrame)
         {
-            initStkLclCnt += varDsc->lvStructGcCount;
+            if (!counted)
+            {
+                initStkLclCnt += varDsc->lvStructGcCount;
+            }
```

Comment:
You might want to set counted to true here.

Reply:
Not really necessary, since the compiler will eliminate it as unused in the remainder of the loop body, but it might be prudent to go ahead and set it, with a comment, in the event that further changes are made at some point (that would cause us to want to query `counted` downstream).